nix/deps/gradle/generate.sh

```diff
@@ -42,22 +42,10 @@ function gen_deps_list() {
 # https://github.com/status-im/status-mobile/issues/15447
 function add_deps_hack() {
     echo -n \
-'com.android.tools.build:gradle:1.3.1
-com.squareup.okio:okio:1.13.0
-com.squareup.okio:okio:1.15.0
-com.squareup.okhttp3:okhttp:3.12.1
```

Comment:
Why does this PR include changes from Sid's PR?

Reply:
There's a rebase issue because the PR now includes a few unrelated commits.

Reply:
not sure why, but I've fixed it